packages/create-portal-app/demo_index.js

```diff
@@ -0,0 +1,9 @@
+const copy = require("./helpers/copy");
```

Comment:
I think this file is no longer needed? What do you think?

Reply:
copy still in use, it copies the portal directory to the new app

Reply:
I mean the file itself `demo_index.js`

Reply:
ok, I will remove the file

Reply:
removed on [this commit](https://github.com/datopian/portal.js/pull/52/commits/96e0659d1a5f2dc70d81d5c3f059053af7c39afd)